draft/oss-support/post.md

```diff
@@ -0,0 +1,70 @@
+# OSS Support
```

Comment:
Maybe think of a few alternate titles for this. Something a little more catchy/intriguing?